common/src/main/java/bisq/common/config/Config.java

```diff
@@ -373,7 +373,7 @@ public Config(String defaultAppName, File defaultUserDataDir, String... args) {
                         .defaultsTo(false);
 
         ArgumentAcceptingOptionSpec<String> providersOpt =
-                parser.accepts(PROVIDERS, "List custom providers")
+                parser.accepts(PROVIDERS, "List custom price providers")
```

Comment:
I've always wanted to rename this option. Improving the description is a fine start.

```suggestion
                parser.accepts(PROVIDERS, "List custom pricenodes")
```